blocks/gmo-campaign-details/gmo-campaign-details.js

```diff
@@ -10,10 +10,23 @@ let blockConfig;
 const programName = getQueryVariable('programName');
 
 export default async function decorate(block) {
+
     const programData = await getProgramInfo(programName, "getProgramDetails");
-    const deliverables = getProgramInfo(programName, "getProgramDeliverables");
+    const deliverables = await getProgramInfo(programName, "getProgramDeliverables");
+
+    const p0TargetMarketArea = programData.data.programList.items[0].p0TargetMarketArea;
+    const p1TargetMarketArea = programData.data.programList.items[0].p1TargetMarketArea;
+
+    //Get unique array of platforms
+    const platforms = [...new Set(programData.data.deliverableList.items
```

Comment:
`getDeliverableScopes` function can be reused to get platforms unique values. May be function needs to be renamed to `getUniqueValues`